### PR TITLE
Use latest instead of tag in Automated Upgrades

### PR DESCRIPTION
**Pull request by elmariofredo**

### CODE REVIEW

**Comment by brandond** [comment]
@dweomer should review this

File: content/k3s/latest/en/upgrades/automated/_index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ To automate upgrades in this manner you must:
 ### Install the system-upgrade-controller
 The system-upgrade-controller can be installed as a deployment into your cluster. The deployment requires a service-account, clusterRoleBinding, and a configmap. To install these components, run the following command:
 ```
-kubectl apply -f https://github.com/rancher/system-upgrade-controller/releases/download/v0.6.2/system-upgrade-controller.yaml
+kubectl apply -f https://github.com/rancher/system-upgrade-controller/releases/latest/download/system-upgrade-controller.yaml
```
Comment (dweomer) [inline content/k3s/latest/en/upgrades/automated/_index.md:36]: I don't want to hold up a documentation PR, especially if it is intended to solve for a customer need, but I am wary of having customers rely on the `latest` convention. I would like to point folks at something like the [k3s upgrader chart](https://github.com/rancher/system-charts/tree/dev-v2.6/charts/rancher-k3s-upgrader) but that is a rancher-specific system chart and likely not appropriate here.

@cjellick @cwayne18 should we seek to publish a chart for SUC?

**Review by dweomer — APPROVED** [review]

File: content/k3s/latest/en/upgrades/automated/_index.md (hunk 1 above)
Comment (cjellick) [reply content/k3s/latest/en/upgrades/automated/_index.md:36]: Do you konw what this sovles jacob? Is there some bug in the older version?

Comment (dweomer) [reply content/k3s/latest/en/upgrades/automated/_index.md:36]: yeah, a big one: https://github.com/rancher/system-upgrade-controller/issues/134

**Pull request merged**